internal/generationtest/generation_test.go

```diff
@@ -51,6 +53,7 @@ func TestFullGeneration(t *testing.T) {
 				GazelleBinaryPath:    absoluteGazelleBinary,
 				BuildInSuffix:        *buildInSuffix,
 				BuildOutSuffix:       *buildOutSuffix,
+				Timeout:              time.Duration(*timeoutSeconds) * time.Second,
```

Comment:
It may make sense to pull this conversion outside of the loop

Reply:
Moved - thanks!